backend/blueprints/spa_api/service_layers/replay/tag.py

```diff
@@ -1,34 +1,53 @@
-from typing import List
-
-from flask import g
+from typing import List, Dict
 
 from backend.blueprints.spa_api.service_layers.utils import with_session
 from backend.utils.global_functions import get_current_user_id
 from ...errors.errors import CalculatedError, TagNotFound
 from backend.database.objects import Tag as DBTag
 from backend.database.wrapper.tag_wrapper import TagWrapper, DBTagNotFound
 
-
 class Tag:
-    def __init__(self, name: str, owner: str):
+    def __init__(self, name: str, owner: str, db_tag: DBTag = None):
+        super().__init__()
         self.name = name
-        self.ownerId = owner
+        self.owner_id = owner
+        self.db_tag = db_tag
+
+    def toJSON(self, with_id=None):
```

Comment:
I have some minor gripes with this function. 1. name seems to not be snake case. 2. i think it will be better named `to_dict`.

Reply:
to_dict is not appropriate because it is not a true dictionary object.

Reply:
? Do you mean it's not exactly equivalent to `__dict__`? I felt to_dict would be right because the returned thing is a dictionary. But it's fine like this.